libyara/modules/pe/pe.c

```diff
@@ -292,6 +292,8 @@ static void pe_parse_debug_directory(
   size_t pdb_path_len;
   char* pdb_path = NULL;
   
+  set_integer(0, pe->object, "is_reproducible_build");
```

Comment:
Does it make sense to set this to zero here? What happens if the debug directory entry (or something else we check later on) is corrupt. In this case a user can get an incorrect value (`pe.is_reproducible_build` would be zero) when it should be `UNDEFINED` at this point.

Reply:
Thank you for your suggestion. Now the value is set to 0 only when debug directory is empty or when code successfully iterated through all entries